package.json

```diff
@@ -92,11 +92,11 @@
     "@aws-sdk/client-kinesis-video": "3.327.0",
     "@aws-sdk/client-kinesis-video-archived-media": "3.327.0",
     "@aws-sdk/client-s3": "3.327.0",
-    "@iot-app-kit/core": "3.3.0",
-    "@iot-app-kit/react-components": "3.3.0",
-    "@iot-app-kit/scene-composer": "3.3.0",
-    "@iot-app-kit/source-iottwinmaker": "3.3.0",
-    "aws-iot-twinmaker-grafana-utils": "file:libs/aws-iot-twinmaker-grafana-utils-1.0.0.tgz",
+    "@iot-app-kit/core": "3.4.0",
+    "@iot-app-kit/react-components": "3.4.0",
+    "@iot-app-kit/scene-composer": "3.4.0",
+    "@iot-app-kit/source-iottwinmaker": "3.4.0",
+    "aws-iot-twinmaker-grafana-utils": "file:libs/aws-iot-twinmaker-grafana-utils-2.0.0.tgz",
```

Comment:
I don't think we want this to be upgraded to 2.0

Reply:
That should be ok..